### PR TITLE
fix: Orchestrator worker crashes if accessing getLabelResolversAsync concurrently

### LINKED ISSUE #6666: Orchestrator: Warmup can crash when trying to access getLabelResolverAsync concurrently
## Describe the bug
The existing code uses a `promise.all` to access `getLabelResolverAsync`, and can cause the Orchestrator worker thread to crash. This is occasionally seen when using Orchestrator with a previously trained multilanguage bot with multiple blu files.

## Version
main 

## Browser
All

## OS
All

### PR DESCRIPTION
**Pull request by taicchoumsft**
## Description
Remove `promise.all` when calling `getLabelResolversAsync`, it occasionally crashes the Orchestrator worker thread.

Exceptions were also not forwarded correctly, when exceptions occurred during build or warmup of Orchestrator, the process never ends since we never send a message back to the main thread. It's also not possible to send the error object in raw form, the process boundary prevents this.  So extract the `message` and `stack` property only and forward those.

## Task Item
fixes #6666

### CODE REVIEW

File: Composer/packages/server/src/models/bot/process/orchestratorWorker.ts [hunk 1]
```diff
@@ -86,26 +86,36 @@ export async function warmUpCache(generatedFolderPath: string, projectId: string
...
-
-      return modelData.model && snapshotData.length
-        ? await Orchestrator.getLabelResolversAsync(modelData.model, '', new Map(snapshotData), false)
-        : new Map<string, LabelResolver>();
-    })
-  );
-
-  cache.set(projectId, new Map([...enMap, ...multilangMap]));
-
+      if (labelResolverMap?.entries) {
```
Comment (hatpick) [inline Composer/packages/server/src/models/bot/process/orchestratorWorker.ts:111]: ```if(labelResolverMap)``` should be enough

File: Composer/packages/server/src/models/bot/process/orchestratorWorker.ts [hunk 2]
```diff
@@ -179,14 +189,10 @@ const handleMessage = async (msg: RequestMsg) => {
       }
     }
   } catch (error) {
-    return { id: msg.id, error };
+    process.send?.({ id: msg.id, error: { message: error?.message, stack: error?.stack } });
```
Comment (hatpick) [inline Composer/packages/server/src/models/bot/process/orchestratorWorker.ts:192]: shouldn't error be empty if there are no error?
```error: error ? { message: error.message, stack: error.stack } : undefined```

Comment (taicchoumsft) [reply Composer/packages/server/src/models/bot/process/orchestratorWorker.ts]: Thanks, addressed

File: Composer/packages/server/src/models/bot/process/orchestratorWorker.ts (hunk 1 above)
Comment (taicchoumsft) [reply Composer/packages/server/src/models/bot/process/orchestratorWorker.ts]: Addressed

**Review by hatpick — APPROVED** [review]

**Pull request merged**